python/lolopy/loloserver.py

```diff
@@ -60,5 +61,5 @@ def get_java_gateway(reuse=True, skip_devel_version=False):
     if _lolopy_gateway is None or not reuse:
         lolo_path = find_lolo_jar(skip_devel_version)
         _lolopy_gateway = JavaGateway.launch_gateway(classpath=os.path.pathsep.join([
-            os.path.abspath(lolo_path)]), die_on_exit=True)
+            os.path.abspath(lolo_path)]), redirect_stdout=sys.stdout, die_on_exit=True)
```

Comment:
Does this do anything?

Reply:
Yes, it means that anything printed to the standard out from lolo gets printed to the output from lolopy. Otherwise, the output from lolo gets routed to /dev/null. Getting the printouts is useful in debugging (e.g., if I put print statements in lolo). 

Besides debugging, the output has no purpose. But, I decided to keep the printing in lolopy so that I don't have to continually add it when I'm debugging.

Reply:
Ahh, I didn't know the default was `/dev/null`.  That output is definitely good to have.